### PR TITLE
Make balance reporting on dashboard consistent 

### PR DESCRIPTION
**Pull request by samsondav**
Make the key list ordering deterministic.

### CODE REVIEW

File: core/web/eth_keys_controller.go [hunk 1]
```diff
@@ -66,6 +67,10 @@ func (ekc *ETHKeysController) Index(c *gin.Context) {
 
 		resources = append(resources, *r)
 	}
+	// Put funding keys to the end
```
Comment (RyanRHall) [inline core/web/eth_keys_controller.go:70]: :+1

File: core/services/keystore/eth.go [hunk 2]
```diff
@@ -78,6 +78,7 @@ func (ks *eth) GetAll() (keys []ethkey.KeyV2, _ error) {
 	for _, key := range ks.keyRing.Eth {
 		keys = append(keys, key)
 	}
+	sort.Slice(keys, func(i, j int) bool { return keys[i].Cmp(keys[j]) < 0 })
```
Comment (RyanRHall) [inline core/services/keystore/eth.go:81]: can we add a test for this?

File: yarn.lock [hunk 3]
```diff
@@ -6450,7 +6450,7 @@ binary-extensions@^2.0.0:
   resolved "https://registry.npmjs.org/binary-extensions/-/binary-extensions-2.0.0.tgz#23c0df14f6a88077f5f986c0d167ec03c3d5537c"
   integrity sha512-Phlt0plgpIIBOGTT/ehfFnbNlfsDEiqmzE2KRXoX1bLIlir4X/MR+zSyBEkL05ffWgnRSf/DXv+WrUAVr93/ow==
 
-bindings@^1.2.1, bindings@^1.5.0:
+bindings@^1.5.0:
   version "1.5.0"
```
Comment (RyanRHall) [inline yarn.lock:6454]: why all the yarn changes?

**Review by RyanRHall — CHANGES_REQUESTED** [review]

File: core/services/keystore/eth.go (hunk 2 above)
Comment (samsondav) [reply core/services/keystore/eth.go:81]: We actually already did have a test here, it was just "accidentally" passing.

File: yarn.lock (hunk 3 above)
Comment (samsondav) [reply yarn.lock:6454]: I just ran `yarn` and it did this.

**Review by RyanRHall — APPROVED** [review]

**Pull request merged**